app/src/main/java/org/example/app/entity/Demo.java

```diff
@@ -0,0 +1,51 @@
+package org.example.app.entity;
+
+import lombok.*;
+import jakarta.persistence.*;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.annotation.ReadOnlyProperty;
+import org.springframework.data.relational.core.mapping.Table;
+import org.springframework.data.relational.core.mapping.Column;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Table(schema = "example", name = "demo")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+public class Demo {
+    @Id
+    @SequenceGenerator(name = "demo_seq", sequenceName = "example.demo_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "demo_seq")
+    private Long id;
+
+    @Column("name")
+    private String name;
+
+    @ReadOnlyProperty
+    @Column("created_at")
+    private LocalDateTime createdAt;
+
+    @Column("updated_at")
+    @LastModifiedDate
+    private LocalDateTime updatedAt;
+
+    public Long getId() {

```

Comment:
To fix Lombok Anntation add `annotationProcessor("org.projectlombok:lombok:1.18.32")` to `build.gradle` file